src/averageAges.js

```diff
@@ -15,6 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const peopleMen = people.filter((person) => person.sex === 'm'
+    && (Math.ceil(person.died / 100) === century || !century));
```

Comment:
It's easier to read in my opinion 
```suggestion
  const peopleMen = people.filter((person) => 
    person.sex === 'm' && 
    (Math.ceil(person.died / 100) === century || !century)
  );
```